cmux-tui/bindings/go/stream.go

```
@@ -408,16 +431,31 @@ func openStream[T any](
 		}
 		return failOpen(err)
 	}
-	opened, err := decodeValue[StreamOpened](raw, operation.Name+" result")
-	if err != nil {
-		return failOpen(err)
+	openedID := StreamID("")
```

golangci-lint (2.12.2)

[error] assigned to openedID, but reassigned without using the value

(wastedassign)